flixel/text/FlxText.hx

```diff
@@ -942,7 +962,7 @@ class FlxText extends FlxSprite
 	{
 		var graphic:BitmapData = _hasBorderAlpha ? _borderPixels : graphic.bitmap;
 		_matrix.translate(x, y);
-		graphic.draw(textField, _matrix);
+		drawTextFieldTo(graphic);
```

Comment:
@Gama11 We can draw the borders without calling `drawTextFieldTo()`, it will draw ~0.5 pixels displaced but the result is not too bad. This would reduce `drawCalls`, but I don't like the idea.

We also could remove the border property and use the `IFlxEffect` here as we need to use with any `FlxSprite`. But the current `FlxOutlineEffect` is more expensive than `FlxTextBorderStyle.OUTLINE` and `FlxTextBorderStyle.OUTLINE_FAST`. I openned an issue to improve `IFlxEffect`s, including outile and also a new one called `FlxShadowEffect`
https://github.com/HaxeFlixel/flixel-addons/issues/228